Update active-directory-b2c-reference-password-complexity.md

dakoer opened the pull request:
Added reference under first heading to advise customer to engage support to enable this feature on their tenant.

azurecla commented:
Hi __@brjohnstmsft__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (brjohnst). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

dakoer committed:
Update active-directory-b2c-reference-password-complexity.md

Added reference under first heading to advise customer to engage support to enable this feature on their tenant.
